core-libs/setup/ssr/optimized-engine/ssr-optimization-options.ts

```diff
@@ -132,13 +133,26 @@ export enum RenderingStrategy {
   ALWAYS_SSR = 1,
 }
 
+export interface RenderingBlocked {
+  urls?: string[];
+  params?: string[];
+}
```

Comment:
`query:` What would you say about changing the interface name to `RenderingStrategyResolverOptions`?
```suggestion
export interface RenderingStrategyResolverOptions {
  excludedUrls?: string[];
  excludedParams?: string[];
}
```

Such names seem to be self-explanatory and in addition, the proposed interface name is a noun which I believe is a good practice.

Btw. I think this interface should be moved to a different place... Doesn't it cause circular dependency?

Reply:
changed